blue-note/theme.json

```diff
@@ -214,6 +214,15 @@
 	},
 	"styles": {
 		"blocks": {
+			"core/button": {
+				"variations": {
+					"outline": {
+						"border": {
+							"width": "1px"
```

Comment:
The default is 2px, I think the design looked more like 1px.

Reply:
The design styles are  0.5px dashed #000 according to figma, but I think 1px is good